### PR TITLE
Users can answer questions which expect a number

### PR DESCRIPTION
**Pull request by LornaHarwood**
## Changes in this PR

- Users can answer questions which expect a number
- An exception is rasied when AnswerFactory receives an unexpected question type

## Screenshots of UI changes
Screen Shot 2021-02-01 at 16 32 07

### CODE REVIEW

**Review by jacksonj04 — CHANGES_REQUESTED** [review]

File: spec/features/visitors/anyone_can_complete_a_journey_spec.rb [hunk 1]
```diff
@@ -45,6 +45,19 @@
       end
     end
 
+    context "when Contentful entry is of type numbers" do
```
[the remark this answers is not in the document]
Comment (LornaHarwood) [reply spec/features/visitors/anyone_can_complete_a_journey_spec.rb:48]: 👍 These have been added

File: app/presenters/number_answer_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class NumberAnswerPresenter < SimpleDelegator
```
Comment (jacksonj04) [inline app/presenters/number_answer_presenter.rb:1]: For completeness we should add a test to confirm this behaves as intended - see `/spec/presenters` for some examples to base it on.

**Review by jacksonj04 — CHANGES_REQUESTED** [review]
One very quick missing test, then I think this one's good to roll!

File: app/presenters/number_answer_presenter.rb (hunk 2 above)
Comment (LornaHarwood) [reply app/presenters/number_answer_presenter.rb:1]: 💯 - I have added in a spec test for `NumberAnswerPresenter`

**Review by jacksonj04 — APPROVED** [review]
🚀

**Pull request merged**